src/main/java/twilight/Task.java

```diff
@@ -0,0 +1,60 @@
+package twilight;
+
+/**
+ * Stores all the information pertaining to a task and allows modification to status.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+//  false status indicates incomplete item
+
+    /**
+     * Creates a Task with status as incomplete.
+     *
+     * @param description What the task is.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Creates a task.
+     *
+     * @param description What the task is.
+     * @param status Whether task is complete.
+     */
+    public Task(String description, boolean status) {
+        this.description = description;
+        this.isDone = status;
+    }
+
+    @Override
+    public String toString() {
+        if (this.isDone) {
+            return "[X] " + this.description;
+        } else {
+            return "[ ] " + this.description;
+        }
+    }
+
+    /**
+     * Returns a string format of the task when storing tasklist in a text file.
+     * 1 and 0 indicate the completion of the task with 1 being complete.
+     */
+    public String toStorage() {
+        if (isDone) {
+            return "1," + this.description;
+        } else {
+            return "0," + this.description;
+        }
+    }
+
+    public void setDone() {
```

Comment:
According to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html): 
> Setter methods for boolean variables must be of the form:
> `void setFound(boolean isFound);`

Perhaps you can lump `setDone` and `setUndone` into one `setDone` method? Or rename the methods to something along the lines of `mark` and `unmark`?